packages/queries/src/types/Schema.ts

```diff
@@ -47,38 +47,46 @@ export interface ColumnDef {
     type: string;
     description: string;
     properties: {
-        type: {
-            id: string;
-            title?: string;
-            type: string;
-            tsType?: string;
-            description: string;
-            minimum?: unknown;
-            maximum?: unknown;
-            format?: string;
-            enum?: readonly string[];
-        };
-        selectable: {
-            type: string;
-            const: boolean;
-        };
-        includable: {
-            type: string;
-            const: boolean;
-        };
-        whereable: {
-            type: string;
-            const: boolean;
-        };
-        nullable: {
-            type: string;
-            const: boolean;
-        };
-        isPrimaryKey: {
-            type: string;
-            const: boolean;
-        };
+        type: ColumnDefTypeDef;
```

Comment:
I'm not sure this is an improvement in readability over the previous version.

`{type:string, const:boolean}` is much easier to read than `ColumnDefAttributeDef`. Seems like a layer of indirection to me.

Reply:
Wasn't much for readablity as it was for reusability (i.e. deduplication)... but if it's severely affecting readability, as can switch it back

Reply:
De-duplication between the fields?

Reply:
Yes.. the selectable, whereable, includeable, nullable and isPrimaryKey fields are all assigned to an identical object: `{ type:string, const:boolean }`, which is what `ColumnDefAttributeDef` is equal to